content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -48,10 +48,11 @@ export class AccessCheckerForWearables {
 
     private async checkCollectionAccess(collection: string, itemId: number, ethAddress: EthAddress): Promise<boolean> {
         try {
-            const collectionItems: WerableCollectionItems = await this.getCollectionItems(collection, itemId, ethAddress)
-            return collectionItems.collections[0]?.creator === ethAddress
-                || collectionItems.collections[0]?.managers.includes(ethAddress)
-                || collectionItems.items[0]?.managers.includes(ethAddress)
+            const ethAddressLowercase = ethAddress.toLocaleLowerCase()
+            const collectionItems: WerableCollectionItems = await this.getCollectionItems(collection, itemId, ethAddressLowercase)
+            return collectionItems.collections[0]?.creator === ethAddressLowercase
```

Comment:
Why are we checking only the `[0]`th element?

Reply:
What I wonder actually is what's the `getCollectionItems` contract interface, why does it need an address, and if it returns an array shouldn't we check all the items?